### PR TITLE
Default to stripping the URL query | spotfix

### PR DESCRIPTION
**Pull request by barryhughes**
Re [conversation here](https://tribe.slack.com/archives/CEZ8ZLHQE/p1555603070190800?thread_ts=1555598317.187100&cid=CEZ8ZLHQE), this elimates the query component of the URL when `getCurrentUrl()` is called. Otherwise, a UTM query will scupper the tests that determine if we are on the homepage, etc.

:ticket: _Spotfix—no ticket required_

### CODE REVIEW

File: dist/frontend.js [hunk 1]
```diff
@@ -12,21 +12,32 @@
...
-
+      var protocolSeparator = url.match(/:\/\//); // Remove the protocol
 
       if (protocolSeparator !== null) {
         url = url.slice(protocolSeparator.index + 3);
       }
 
+      var querySeparator = url.match(/\?/); // Remove the query
+
+      if (!includeQuery && querySeparator !== null) {
```
Comment (geoffgraham) [inline dist/frontend.js]: A little spacing in the parenthesis.

File: dist/frontend.js [hunk 2]
```diff
@@ -12,21 +12,32 @@
...
       var url = document.location.toString();
-      var protocolSeparator = url.match(/:\/\//); // Remove trailing slash for uniformity
-
-      if ('/' === url.slice(-1)) {
-        url = url.slice(0, -1);
-      } // Remove the protocol
-
+      var protocolSeparator = url.match(/:\/\//); // Remove the protocol
 
       if (protocolSeparator !== null) {
```
Comment (geoffgraham) [inline dist/frontend.js]: Spacing here.

File: dist/frontend.js [hunk 3]
```diff
@@ -12,21 +12,32 @@
...
         url = url.slice(protocolSeparator.index + 3);
       }
 
+      var querySeparator = url.match(/\?/); // Remove the query
+
+      if (!includeQuery && querySeparator !== null) {
+        console.log(querySeparator.index);
```
Comment (geoffgraham) [inline dist/frontend.js]: 🎵 Sp-sp-sp-spaces 🎵

File: dist/frontend.js [hunk 4]
```diff
@@ -12,21 +12,32 @@
...
+        url = url.slice(0, querySeparator.index);
```
Comment (geoffgraham) [inline dist/frontend.js]: 🌎 Inner space

File: dist/frontend.js [hunk 5]
```diff
@@ -12,21 +12,32 @@
...
+      } // Remove trailing slash for uniformity
+
+
+      if ('/' === url.slice(-1)) {
```
Comment (geoffgraham) [inline dist/frontend.js]: S P A C E... the final frontier.

File: dist/frontend.js (hunk 1 above)
Comment (barryhughes) [reply dist/frontend.js]: This is actually in `dist/` (so is the compiled code and not the source), we don't need to worry about formatting here.

File: dist/frontend.js (hunk 2 above)
Comment (barryhughes) [reply dist/frontend.js]: Same as earlier - this is generated from the source and is not the source itself.

File: dist/frontend.js (hunk 3 above)
Comment (barryhughes) [reply dist/frontend.js]: As earlier :-)

File: dist/frontend.js (hunk 4 above)
Comment (barryhughes) [reply dist/frontend.js]: As earlier :-)

File: dist/frontend.js (hunk 5 above)
Comment (barryhughes) [reply dist/frontend.js]: As earlier :-)

**Pull request merged**